lib/common.ts

```diff
@@ -0,0 +1,48 @@
+import { BalenaSDK } from 'balena-sdk';
```

Comment:
* Regarding the name of this file, `lib/common.ts`, it sounds very generic, while the contents are quite specific to parsing a flag (`--release-tag`) that applies to two CLI commands only. Instead of creating a new file, what about using one of the following existing files?
  * `lib/utils/common-flags.ts` (though currently it contains the flags themselves, rather than functions to parse them)
  * `lib/utils/compose_ts.ts` - this file contains `composeCliFlags` at the end, and the file is already imported by both `push.ts` (line 279 in master, inside the `run()` method) and `deploy.ts`. (The code in `compose_ts.ts` is mainly used by the build and deploy commands, and `--release-tag`.)

* If a new file is created, we usually add the Apache licence comment at the top.